src/Lykke.HttpClientGenerator/HttpClientGenerator.cs

```diff
@@ -35,6 +36,13 @@ public static HttpClientGeneratorBuilder BuildForUrl(string rootUrl)
             return new HttpClientGeneratorBuilder(rootUrl);
         }
 
+        /// <inheritdoc />
+        public HttpClientGenerator(string rootUrl, IEnumerable<ICallsWrapper> callsWrappers,
+            IEnumerable<DelegatingHandler> httpDelegatingHandlers, TimeSpan? timeout = null)
```

Comment:
Make timeout not optional. At the moment this ctor is ambigous with ctor at line 47 when timeoute is not specified

Reply:
And why timeout parameter is not used?

Reply:
I've found out that this ctor is not used at all, remove it.